cli/pcluster/config/validators.py

```diff
@@ -1253,3 +1253,32 @@ def fsx_ignored_parameters_validator(section_key, section_label, pcluster_config
 def _describe_ec2_key_pair(key_pair_name):
     """Return information about the provided ec2 key pair."""
     return boto3.client("ec2").describe_key_pairs(KeyNames=[key_pair_name])
+
+
+def ebs_volume_iops_validator(section_key, section_label, pcluster_config):
+    errors = []
+    warnings = []
+
+    section = pcluster_config.get_section(section_key, section_label)
+    volume_size = section.get_param_value("volume_size")
+    volume_type = section.get_param_value("volume_type")
+    volume_type_to_iops_ratio = {"io1": 50, "io2": 500}
+    if section.get_param_value("volume_iops"):
+        volume_iops = section.get_param_value("volume_iops")
```

Comment:
I would avoid this conditional. Always set `volume_iops` regardless of what `get_param_value` returns. Then instead of checking whether the parameter is set, you can check that in the conditional you use to decide whether or not to test the values.

```python
if volume_type in volume_type_to_iops_ration.keys() and volume_iops:
  # validation logic
```

Reply:
Done